setup.py

```diff
@@ -55,15 +53,15 @@
 }
 
 INSTALL_REQUIRES = [
-    'aodncore>=0.31.0',
+    'aodncore>=1.0.0',
     'cc-plugin-imos>=1.3.0',
     'matplotlib==1.5.1',
     'aodntools>=0.4.4',
```

Comment:
Ooops, one last thing: should we change this dependency to `aodntools>=1.0.0` too?